Fail build with run-android when updating from 2.0.7 to 2.0.8

<!---

Questions: If you have questions about HOW TO use Realm, please ask on
StackOverflow: http://stackoverflow.com/questions/ask?tags=realm
We monitor the `realm` tag.

Feature Request: Just fill in the first two sections below.

Bugs: To help you as fast as possible with an issue please describe your issue
and the steps you have taken to reproduce it in as much detail as possible.

-->

## Goals

I am trying to run a react-native app that was previously working with 2.0.7

## Expected Results

react-native run-android works

## Actual Results

react-native run-android fails

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Cannot evaluate module realm : Configuration with name 'default' not found.
```

## Steps to Reproduce

<!--- What are steps we can follow to reproduce this issue? -->

## Code Sample

This is the project: https://github.com/status-im/status-react

## Version of Realm and Tooling

- Realm JS SDK Version: realm 2.0.8
- Node or React Native: - Client OS & Version: 0.43.4
- Which debugger for React Native: None

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,17 @@
+X.Y.Z Release notes
+=============================================================
+### Breaking changes
+* None.
+
+### Enchancements
+* Reenable Realm for RN Android (#1506).
+
+### Bug fixes
+* None.
+
+### Internal
+* None.
+
 2.0.8 Release notes (2017-11-17)
 =============================================================
 ### Breaking changes

